ql/src/java/org/apache/hadoop/hive/ql/parse/repl/dump/events/AlterTableHandler.java

```diff
@@ -147,16 +197,19 @@ public void handle(Context withinContext) throws Exception {
 
     Table qlMdTableBefore = new Table(before);
     Set<String> bootstrapTableList;
+    ReplScope oldReplScope;
     if (Scenario.RENAME == scenario) {
       // Handling for table level replication is done in handleForTableLevelReplication method.
```

Comment:
handleForTableLevelReplication doesn't handle rename flow. Incorrect comment.

Reply:
code is removed